quaternions/quaternions.py

```diff
@@ -1,7 +1,7 @@
 import functools
 import numpy as np
 from collections import Iterable
-
+from quaternions.utils import (covariance_matrix_from_angles, sigma_lerner, orthogonal_matrix)
```

Comment:
No need to have parentheses here

Reply:
yes, but i was @matiasg  style to put then anyway

Reply:
parentheses there are needed when you need to split into multiple lines. Otherwise, they are not.
I'm not saying that I always go by this rule, but it's the best rule I can produce ;-)